tests/Extensions/TimedScopes.UnitTests/ActivityObserversIntializerTests.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System.Diagnostics;
+using System.Linq;
+using System.Threading;
+using Microsoft.Omex.Extensions.Abstractions.Activities.Processing;
+using Microsoft.Omex.Extensions.TimedScopes;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+
+namespace Hosting.Services.UnitTests
+{
+	[TestClass]
+	public class ActivityObserversIntializerTests
+	{
+		[TestMethod]
+		public void ActivityObserversIvokedProperly()
+		{
+			string name = nameof(ActivityObserversIvokedProperly);
+			Mock<IActivityStartObserver> startObserver = new Mock<IActivityStartObserver>();
+			Mock<IActivityStopObserver> stopObserver = new Mock<IActivityStopObserver>();
+			ActivityObserversIntializer initializer = new ActivityObserversIntializer(
+				new [] { startObserver.Object },
+				new [] { stopObserver.Object });
+
+			try
+			{
+				initializer.StartAsync(CancellationToken.None);
```

Comment:
>StartAsync [](start = 16, length = 10)

Do you wish to `await` this? I realise your code doesn't require it, but it seems like someone could potentially change the implementation in the future so that having an `await` here would make things simpler

Reply:
Make sense, thank you, fixed